### PR TITLE
Added effects "Detach Random Wheel" and "Disassemble Current Vehicle"

### PR DESCRIPTION
**Pull request by Reguas**
& assets for entity fragments
Screenshot from 2022-06-13 01-57-48

Fixes https://github.com/gta-chaos-mod/ChaosModV/issues/107

### CODE REVIEW

**Comment by OnlyRealNubs** [comment]
Looks nice! I envy the work you did here.

**Comment by Reguas** [comment]
Well, most of it was copy-pasting the code from SHVDotNet source xD

**Comment by OnlyRealNubs** [comment]
> Well, most of it was copy-pasting the code from SHVDotNet source xD

Well still, well done lol

File: ChaosMod/Effects/db/Vehs/VehsDetachWheel.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
+Effect by Reguas
+*/
+
+#include "Memory/EntityFragments.h"
+#include <stdafx.h>
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Vehs/VehsDetachWheel.cpp:6]: Should be the first include, add an empty line between both to make sure they don't get reordered by the formatter

```suggestion
#include <stdafx.h>

#include "Memory/EntityFragments.h"
```

File: ChaosMod/Effects/db/Vehs/VehsDetachWheel.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	}
+}
+
+// clang-format off
+REGISTER_EFFECT(OnStart, nullptr, nullptr, EffectInfo
+{
+.Name = "Detach Random Wheel",
+.Id = "vehs_detach_wheel"
+}
+);
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Vehs/VehsDetachWheel.cpp:41]: Indents are missing here

File: ChaosMod/Effects/db/Vehs/VehsDisassemble.cpp [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/*
+    Effect by Reguas
+*/
+
+#include "Memory/EntityFragments.h"
+#include <stdafx.h>
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Vehs/VehsDisassemble.cpp:6]: Same applies here

File: ChaosMod/Effects/db/Vehs/VehsDisassemble.cpp [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	}
+}
+
+// clang-format off
+REGISTER_EFFECT(OnStart, nullptr, nullptr, EffectInfo
+{
+	.Name = "Disassemble Current Vehicle",
+	.Id = "vehs_disassemble"
+}
+);
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Vehs/VehsDisassemble.cpp:36]: Here too

**Review by pongo1231 — CHANGES_REQUESTED** [review]

**Pull request merged**